### PR TITLE
[BEAM-3993] read gitignore and add it in rat exclusions

### PR DESCRIPTION
**Pull request by rmannibucau**
rat build doesn't pass with gradle cause maven merges gitignore in rat exclusions but not ant/gradle

this PR reads gitignore to merge it as well (instead of duplicating them)

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/projects/BEAM/issues/) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand:
   - [ ] What the pull request does
   - [ ] Why it does it
   - [ ] How it does it
   - [ ] Why this approach
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

### CODE REVIEW

**Comment by angoenka** [comment]
R: @lukecwik

File: build.gradle [hunk 1]
```diff
@@ -107,7 +104,6 @@ rat {
     "sdks/python/NOTICE",
```
Comment (lukecwik) [inline build.gradle]: I cleaned up the duplication in https://github.com/rmannibucau/incubator-beam/pull/1, please consider merging into your branch.

**Comment by rmannibucau** [comment]
@lukecwik applied, only "?" I have is why using println instead of the logger which prevents to use --info to control the logs but not a blocker at all.

**Comment by lukecwik** [comment]
Just to get visibility during the build process so that users don't have to type --info. The rest of the build uses it at important points. We can swap to using logging to cut down verbosity at any point in time in the future.

**Pull request merged**